# Section: NETWORK CONFIGURATION
# Reference: broker1 (all other nodes diffed against this)

## broker1 vs broker2
--- broker1
+++ broker2
@@ -1,5 +1,5 @@
 --- Network interfaces ---
-ens3             UP             10.0.11.197/24 metric 512 fe80::109e:bbff:fe5b:1c17/64 
+ens3             UP             10.0.12.138/24 metric 512 fe80::8ff:f0ff:fe67:4fcf/64 
 docker0          DOWN           172.17.0.1/16 
 
 --- ENA driver version ---
@@ -12,7 +12,7 @@
 (ring buffer info unavailable)
 
 --- Peer RTT (inter-node ping sweep) ---
-BROKER_2_IP (10.0.12.138): min rtt = unreachable ms
+BROKER_1_IP (10.0.11.197): min rtt = unreachable ms
 BROKER_3_IP (10.0.13.95): min rtt = unreachable ms
 CONNECT_1_IP (10.0.11.113): min rtt = unreachable ms
 MONITOR_1_IP (10.0.12.184): min rtt = unreachable ms

## broker1 vs broker3
--- broker1
+++ broker3
@@ -1,5 +1,5 @@
 --- Network interfaces ---
-ens3             UP             10.0.11.197/24 metric 512 fe80::109e:bbff:fe5b:1c17/64 
+ens3             UP             10.0.13.95/24 metric 512 fe80::cf2:5cff:feec:c8ed/64 
 docker0          DOWN           172.17.0.1/16 
 
 --- ENA driver version ---
@@ -12,8 +12,8 @@
 (ring buffer info unavailable)
 
 --- Peer RTT (inter-node ping sweep) ---
+BROKER_1_IP (10.0.11.197): min rtt = unreachable ms
 BROKER_2_IP (10.0.12.138): min rtt = unreachable ms
-BROKER_3_IP (10.0.13.95): min rtt = unreachable ms
 CONNECT_1_IP (10.0.11.113): min rtt = unreachable ms
 MONITOR_1_IP (10.0.12.184): min rtt = unreachable ms
 

## broker1 vs connect
--- broker1
+++ connect
@@ -1,6 +1,6 @@
 --- Network interfaces ---
-ens3             UP             10.0.11.197/24 metric 512 fe80::109e:bbff:fe5b:1c17/64 
-docker0          DOWN           172.17.0.1/16 
+ens5             UP             10.0.11.113/24 metric 512 fe80::102e:99ff:fe7d:7093/64 
+docker0          DOWN           172.17.0.1/16 fe80::42:74ff:fe5c:3594/64 
 
 --- ENA driver version ---
 (ethtool unavailable)
@@ -12,8 +12,8 @@
 (ring buffer info unavailable)
 
 --- Peer RTT (inter-node ping sweep) ---
+BROKER_1_IP (10.0.11.197): min rtt = unreachable ms
 BROKER_2_IP (10.0.12.138): min rtt = unreachable ms
 BROKER_3_IP (10.0.13.95): min rtt = unreachable ms
-CONNECT_1_IP (10.0.11.113): min rtt = unreachable ms
 MONITOR_1_IP (10.0.12.184): min rtt = unreachable ms
 

## broker1 vs monitor
--- broker1
+++ monitor
@@ -1,5 +1,5 @@
 --- Network interfaces ---
-ens3             UP             10.0.11.197/24 metric 512 fe80::109e:bbff:fe5b:1c17/64 
+ens5             UP             10.0.12.184/24 metric 512 fe80::8ff:cbff:fed4:a601/64 
 docker0          DOWN           172.17.0.1/16 
 
 --- ENA driver version ---
@@ -12,8 +12,8 @@
 (ring buffer info unavailable)
 
 --- Peer RTT (inter-node ping sweep) ---
+BROKER_1_IP (10.0.11.197): min rtt = unreachable ms
 BROKER_2_IP (10.0.12.138): min rtt = unreachable ms
 BROKER_3_IP (10.0.13.95): min rtt = unreachable ms
 CONNECT_1_IP (10.0.11.113): min rtt = unreachable ms
-MONITOR_1_IP (10.0.12.184): min rtt = unreachable ms
 

